pysatSpaceWeather/instruments/sw_kp.py

```diff
@@ -62,7 +62,6 @@
 import sys
 
 import pysat
-from pysat.utils.time import parse_date
 
```

Comment:
Kp data and metadata look good. Single, multiple, all days. Checked for fill data and didn't find any.
![image](https://user-images.githubusercontent.com/3595641/118724026-ba920600-b7f3-11eb-9e2a-5c923953dd98.png)

'forecast' and 'recent' have similar interface with `inst.load()` and multi-day loading.

Reply:
I can't put the comment at the code line. Fill value for 'forecast' is -1. Recommend np.nan.